CHANGELOG.rst

```diff
@@ -3,7 +3,7 @@ Changelog
 
 .. _section-1:
 
-0.3.0
```

Comment:
You need to add a new heading, and put your changes under that, not delete the old heading.